googlecloudprofiler/client.py

```diff
@@ -257,12 +258,19 @@ def _build_service(self):
     """Builds a discovery client for talking to the Profiler."""
     http = httplib2.Http()
     http = google_auth_httplib2.AuthorizedHttp(self._credentials, http)
+
+    def request_builder(*args, **kwargs):
+      if self._agent_version is not None:
+        headers = kwargs.setdefault('headers', {})
+        headers['X-Goog-API-Client'] = 'profiler-gccl/' + self._agent_version
```

Comment:
self._agent_version should be str(self._agent_version) to handle None. Or in Init, set self._agent_version to 'NO_VERSION' or something descriptive.